azure-pipelines.yml

```diff
@@ -2,185 +2,36 @@ trigger:
-    condition: and(ne(variables['Build.Reason'], 'PullRequest'), contains(variables['IMAGE_NAME'], 'win'))
-    displayName: 'Download certificate on Windows'
-
-  - script: |
-      set CSC_LINK=$(Agent.TempDirectory)\NORDIC_SIGNING_CERTIFICATE.pfx
-      set CSC_KEY_PASSWORD=$(NORDIC_SIGNING_CERTIFICATE_PASSWORD_PFX)
-      npx electron-builder -p never --windows nsis:x64
-    condition: and(ne(variables['Build.Reason'], 'PullRequest'), contains(variables['IMAGE_NAME'], 'win'))
-    displayName: 'Build electron installer on Windows'
-
-  - task: ArchiveFiles@2
-    inputs:
-      rootFolderOrFile: '$(Build.SourcesDirectory)/release/win-unpacked'
-      includeRootFolder: false
-      archiveType: 'zip'
-      archiveFile: '$(Build.ArtifactStagingDirectory)/nrfconnect-portable-$(Build.BuildNumber).zip'
-    condition: and(
-      ne(variables['Build.Reason'], 'PullRequest'),
-      contains(variables['IMAGE_NAME'], 'win'))
-    displayName: 'Pack on Windows x64'
-
-  - task: DownloadSecureFile@1
-    inputs:
-      secureFile: 'NORDIC_SIGNING_CERTIFICATE.p12'
-    condition: and(ne(variables['Build.Reason'], 'PullRequest'), contains(variables['IMAGE_NAME'], 'mac'))
-    displayName: 'Download certificate on macOS'
-
-  - bash: |
-      set -o errexit -o pipefail
-      export CSC_LINK=$(Agent.TempDirectory)/NORDIC_SIGNING_CERTIFICATE.p12
-      export CSC_KEY_PASSWORD=$(NORDIC_SIGNING_CERTIFICATE_PASSWORD_P12)
-      export APPLEID=$(WAYLAND_APPLE_ID)
-      export APPLEIDPASS=$(WAYLAND_APPLE_APP_SPECIFIC)
-      rm -f node_modules/pc-nrfjprog-js/nrfjprog/._*
-      rm -f node_modules/pc-nrfjprog-js/build/Release/._*
-      npx electron-builder -p never
-    condition: and(ne(variables['Build.Reason'], 'PullRequest'), contains(variables['IMAGE_NAME'], 'mac'))
-    displayName: 'Release on macOS'
-
-  - bash: |
-      set -o errexit -o pipefail
-      cp release/*.exe "$(Build.ArtifactStagingDirectory)" || :
-      cp release/*.zip "$(Build.ArtifactStagingDirectory)" || :
-      cp release/*.dmg "$(Build.ArtifactStagingDirectory)" || :
-      cp release/*.tar.gz "$(Build.ArtifactStagingDirectory)" || :
-      cp release/*.AppImage "$(Build.ArtifactStagingDirectory)" || :
-      cp release/latest* "$(Build.ArtifactStagingDirectory)"
-    condition: ne(variables['Build.Reason'], 'PullRequest')
-    displayName: 'Copy artifacts'
-
-  - task: PublishPipelineArtifact@1
-    inputs:
-      targetPath: $(Build.ArtifactStagingDirectory)
-      publishLocation: 'pipeline'
-    displayName: 'Publishing artifacts to artifact storage'
+    value: 1
+  - name: npm_config_cache
+    value: $(Pipeline.Workspace)/.npm
+
+jobs: 
+  - template: pipelines/pipeline.yml
+    parameters: 
+      platform: darwin
+      pool:
+        vmImage: macos-11
+  # - template: pipelines/pipeline.yml
+  #   parameters: 
+  #     platform: darwin_arm
+  #     pool:
+  #       name: wayland_pool_sw_only
+  #       demands:
+  #       - agent.name -equals wl-node-test-17
```

Comment:
What is the reason for this to be commented out?

Reply:
It is a specific build server that I hope we enable in the near future, and the pool setup is a bit specific. I considered puting this in confluence or some other place, but I suspect that wouldn't get checked by the "next" guy trying to set this up. I will try to find another place to "document" how to use the mac arm ci server for this setup :)